### DISCUSSION TITLE
Plain Craft Launcher 历史（大雾丨正在施工）

### DISCUSSION THREAD

**Discussion by PlayMcBKuwu** [Meloong-Git/PCL#3111] [讨论]
- 灵感来源于 [PCL2的前身——PCL1长啥样？](https://www.bilibili.com/video/BV1uQ4y1E79L/)
- 这次不会像 #2132 那么过火了 www 
并且那个 Disscussion 也不打算发布其他的信息了 =w=
- 只是简单的整理一下 PCL 的历史 ~~因为龙猫是一点没在 PCL 历史这方面在石墨或者知乎发文档~~
- **仍在施工！** 欢迎参与补充其他信息（雾）

# Plain Craft Launcher 1 时代
- [龙猫早已开源 www](https://github.com/LTCatt/PCL1) 需要手动编译
- ~~[Get it on 某不知名软件站 1](https://www.jyszcg.com/soft/17410.html) or [Get it on 某不知名软件站 2](https://www.sibspress.org/soft/17410.html)~~
- **[ 等待补充 ]** 没有详细历史 :(
# Plain Craft Launcher 2 时代
- [目前唯一可得知的历史也就 TZG 和 Hiper 了](https://www.bilibili.com/read/cv24602954/)
## PCL2 黑暗时代（TZG 事件时）前中后还发生了什么？
- [PCL2 内测版被到处传播（本帖中群号均已失效）](https://tieba.baidu.com/p/6877222401)
- [和解了](https://www.zhihu.com/people/LTCat/pins)……或许？（这应该是我对这件事的最后两个补充，之后应该不会再发布别的了）
# 等待补充 ing……
***
*如有冒犯，请立即联系我关闭本 Discussion。*

**Comment by iPodtouch6557** [comment]
哦哦哦，我的PCL1就在这个网站下的，后来永远的找不到网址了( )现在找到了

**Reply by PlayMcBKuwu** [reply]
染鹅 PCL1 兼容性很不好就是了（
——Sent on FydeOS PC
